packages/algoliasearch/src/builds/node.ts

```diff
@@ -196,6 +196,148 @@ import { createUserAgent, RequestOptions } from '@algolia/transporter';
 
```

Comment:
Can't we just do `export * from '@algolia/client-*'` ? 
Types listing seems unnecessary and hard to maintain + we would benefits from exporting helpers too

Reply:
(for `node.ts` only obviously)

Reply:
seems like a good option, would you want to make a PR to take this over @bodinsamuel ? The only thing to be careful of with export * is that in some cases types are exported from JS files and then will have runtime errors when accessed directly from JS. I don't think that's really an issue here, but just to keep in mind